grafana-plugin/src/pages/integration/Integration.tsx

```diff
@@ -697,6 +692,17 @@ class Integration extends React.Component<IntegrationProps, IntegrationState> {
       })
       .finally(() => this.setState({ isLoading: false }));
   }
+
+  async loadContactPointsMaybe(id: AlertReceiveChannel['id']) {
+    const { alertReceiveChannelStore } = this.props.store;
```

Comment:
the page will not wait until contact points are loaded, but instead render them when they're available so that we don't slow down the page load.